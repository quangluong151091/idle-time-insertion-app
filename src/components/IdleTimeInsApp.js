import React from 'react';
import Header from './Header';
import SearchBox from './SearchBox';
import { BrowserRouter } from "react-router-dom";
import Routes from '../routers/AppRouter';

class IdleTimeInsApp extends React.Component {
    render() {
        return (
            <div>
                <Header />
                <SearchBox searchBoxName={"userNameSearch"} onSearchTermChange={this.onSearch}/>

                <BrowserRouter>
                    <Routes/>
                </BrowserRouter>
            </div>
        );
    }
}

export default IdleTimeInsApp;