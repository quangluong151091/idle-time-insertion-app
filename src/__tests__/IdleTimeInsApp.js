import React from 'react';
import { shallow } from 'enzyme';

import IdleTimeInsApp from '../components/IdleTimeInsApp';

describe('IdleTimeInsApp', () => {
  it('should render correctly', () => {

    const component = shallow(<IdleTimeInsApp />);
  
    expect(component).toMatchSnapshot();
  });
});