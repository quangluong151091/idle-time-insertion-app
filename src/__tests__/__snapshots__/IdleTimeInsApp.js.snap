// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IdleTimeInsApp should render correctly 1`] = `
<div>
  <Header />
  <SearchBar
    searchBoxName="userNameSearch"
  />
  <BrowserRouter>
    <AppRouter />
  </BrowserRouter>
</div>
`;
